app/client/models/DocPageModel.ts

```diff
@@ -266,7 +266,7 @@ export class DocPageModelImpl extends Disposable implements DocPageModel {
         explanation: (
           isDocOwner
             ? t("You can try reloading the document, or using recovery mode. \
-              Recovery mode opens the document to be fully accessible to owners, and inaccessible to others. \
+Recovery mode opens the document to be fully accessible to owners, and inaccessible to others. \
```

Comment:
Just to be clear, this is the whitespace change that mattered. It results in `a                     weird                  string               for               translators` to translate. The other whitespace changes, and undoing them here, are tolerable but undesirable.

Reply:
We've talked about this on and off over the years. So far we've gone with the solution of "follow the local convention in the part of the codebase you're working on". It has generally worked well enough, with very quick adaptation by new contributors. More recently I've found myself having to waste more time than usual with lint issues so maybe it is time to reconsider. The transition would a somewhat daunting process though.

Reply:
I opened this issue to discuss of the proposal: #595 

> The transition would a somewhat daunting process though.

The right moment may be when there is not much PR to review to avoid conflicts.

Reply:
To add to the fun we have a monorepo for our saas of which grist-core is just a subdirectory sync'd via git subtree, and that repo has its own tooling (phabricator, jenkins, etc) and set of pull requests.